e2e/support/config.js

```diff
@@ -213,6 +213,8 @@ const crossVersionTargetConfig = {
 
 const stressTestConfig = {
   ...defaultConfig,
+  viewportHeight: 800,
```

Comment:
the stress test workflow has been using the [default height value](https://docs.cypress.io/api/commands/viewport#Default-sizing) which is `660px` height.

This is the reason these stress test runs always fail.

- [PR stress tests](https://github.com/metabase/metabase/actions/runs/9368575231)
- [Master stress tests](https://github.com/metabase/metabase/actions/runs/9368577269)

Reply:
Nit:
WDYT about moving both the `viewportHeight` and `viewportWidth` to the `defaultConfig` instead?

Reply:
snapshotConfig is not related to visual regression testing.
The snapshots in this context are the database snapshots (dumps).

Reply:
@nemanjaglumac Thanks for the confirmation. I'll move this up to `defaultConfig` as suggested 🙏

Reply:
The [stress tests passed](https://github.com/metabase/metabase/actions/runs/9379158210/job/25823623995) 💪